### PR TITLE
[webflux] Re-enable blockhound and whitelist BoundedElasticScheduler.schedule

### LINKED ISSUE #11599: Re-enable BlockHound once blocking calls are fixed
##### **Overview of the feature request**

BlockHound has been added in https://github.com/jhipster/generator-jhipster/pull/11009 but it has unveiled some blocking calls in frameworks that we're using and thus our CI tests have become very unstable since.
I plan to temporarily disable BlockHound in order to have trustable integration tests again, but we should re-enable it once the blocking calls are resolved.

### PR DESCRIPTION
**Pull request by murdos**
Fixes #11599

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs [hunk 1]
```diff
@@ -0,0 +1 @@
+<%= packageName %>.config.JHipsterBlockHoundIntegration
```
Comment (cbornet) [inline generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs:1]: Maybe use [`@AutoService`](https://github.com/reactor/BlockHound/blob/master/docs/custom_integrations.md#custom-integrations) ?

Comment (murdos) [reply generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs:1]: I'm not sure adding yet another dependency and another annotation processor is worth the value here, for something that is supposed to be a temporary workaround.

Comment (cbornet) [reply generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs:1]: Adding a file that will need to be cleaned up later doesn't seem better.

Comment (murdos) [reply generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs:1]: Does really cleaning one file (`JHipsterBlockHoundIntegration.java`) or two files (`JHipsterBlockHoundIntegration.java` + meta-inf service declaration) makes a difference?

On the other hand, we have to deal with adding a new dependency (`com.google.auto.service:auto-service-annotations`), a new annotation processor (`com.google.auto.service:auto-service`) and a new property for the version that will have to be updated from time to time, for 2 build systems (gradle and maven).

I'm really not convinced here, in this context, with the use of `@Autoservice`, but I can add it others think it's the better solution.
@jhipster/developers what do you think?

Comment (mraible) [reply generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs:1]: Tests pass - ship it! 🚢

**Comment by cbornet** [comment]
LGTM. Feel free to merge @murdos

**Comment by mshima** [comment]
@murdos webflux started failing with this PR.

Suddenly webflux tests started failing at https://github.com/jhipster/generator-jhipster/pull/13021 with a minor change and I had now clue why.
MariaDB just timeout without no useful error.
Changing sqfull from MariaDB to postgres shows the error.

Merged https://github.com/jhipster/generator-jhipster/pull/13031 to confirm.

**Comment by murdos** [comment]
Bounty claimed: https://opencollective.com/generator-jhipster/expenses/59178

**Comment by pascalgrimaud** [comment]
@murdos : approved

**Pull request merged**